market_next_door_api/market_next_door_api/settings.py

```diff
@@ -76,15 +76,11 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.postgresql',
-        'NAME': 'market_next_door_db',
-        'USER': 'dylantimmons',
-        'HOST': 'localhost', 
-        'PORT': '5432',        
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': BASE_DIR / 'db.sqlite3',
```

Comment:
Thanks for changing it back to sqlite3 - we can double check if we need postgresql for production but this will be good for development.